.github/workflows/imgbot.yml

```
@@ -0,0 +1,22 @@
+name: imgbot Image Optimization
+
+on: pull_request
+
+permissions:
+  pull-requests: write
+
+jobs:
+  approveAndMerge:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'imgbot[bot]' }}
+    steps:
+      - name: Approve imgbot PR
+        run: gh pr review --approve "$PR_URL"
```

Not sure if this is needed, but I've seen this done like this before:

```suggestion
        run: gh pr review --approve "${{ env.PR_URL }}"
```

and an advantage of that is that the variable is expanded to plain text before the workflow runs so you can see the command with variable expanded in the logs